lib/linguist/languages.yml

```diff
@@ -1045,6 +1045,15 @@ DTrace:
   codemirror_mode: clike
   codemirror_mime_type: text/x-csrc
   language_id: 85
+Daml:
```

Comment:
Any reason for choosing "Daml" for the language name (instead of DAML) when everything on official the site you've referenced calls the language "DAML"?

Reply:
Good point, let me get back to you.